backend/eventd/logger.go

```diff
@@ -138,25 +166,46 @@ func (l *rawLogger) ringBuffer() {
 
 	for v := range l.input {
 		select {
-		case l.output <- v:
+		case l.encoderInput <- v:
 		case <-time.After(l.wait):
-			// The new event could not be placed on the outgoing channel, therefore
-			// take the oldest event in the buffer, drop it, and place the new event
-			// at its place
-			<-l.output
-			l.output <- v
+			dropped := false
+			select {
```

Comment:
We had discussed there could be unwanted dropped events. Did you want to rework that logic in a separate fix?

Reply:
Yes, if we rework this I think it needs a separate effort. :+1: